.gitignore

```diff
@@ -77,8 +77,6 @@ local.properties
 *.log
 
 # Build results
-[Dd]ebug/
-[Rr]elease/
```

Comment:
Well I use them. But I guess I could switch to one of the other build directories that we ignore.